src/utils/package-manager.ts

```diff
@@ -24,6 +25,26 @@ export const BUN: PackageManager = {
   buildCommand: 'bun run build',
   runScriptCommand: 'bun run',
   flags: '',
+  detect: () => fs.existsSync(path.join(process.cwd(), BUN.lockFile)),
+};
+export const YARN_CLASSIC: PackageManager = {
+  name: 'yarn',
```

Comment:
Let's not call this "yarn classic" as nobody knows what the hell that is 😂 Instead I would say something like "yarn v1" and "yarn v2/v3/v4" or "yarn (>= version 2)"

Reply:
Yep, make sense.